lms/djangoapps/teams/docs/decisions/0001-field-to-store-enabled-disabled-status.rst

```diff
@@ -0,0 +1,49 @@
+Add a new field to store enable/disabled status of teams
+--------------------------------------------------------
+
+Status
+======
+
+Proposal
+
+
+Context
+=======
+
+While implementing a UI to enable/disable teams we ran into issues due to the
+way teams is considered enabled or disabled, which is with the presence of one
+or more teamsets. So enabling requires adding a teamset and disabling requires
+removing all teamsets.
+
+Since we'd like to have a simple toggle to enable/disable teams this complicates
+the implementation significantly.
+
+Decisions
+=========
+
+We can add a new field called ``teams_enabled`` (or simply ``enabled``) to the
```

Comment:
How do you plan to handle the case where teams is enabled but no team set exists?

Reply:
That is essentially teams in an unconfigured state, so the UI can show show the error condition, and explain that enabling teams needs a teamset to be configured.